website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-dashboard-status-tiles.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Dashboard Status Tiles"
+id: "cloud-dashboard-status-tiles"
+---
+
+## Overview
+In dbt Cloud, the [Metadata API](https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview) can power Dashboard Status Tiles.  A Dashboard Status Tile is placed on a dashboard (specifically: anywhere you can embed arbitrary HTML) to give insight into the quality and freshness of the data feeding into that dashboard. This is done via dbt [exposures](https://docs.getdbt.com/docs/building-a-dbt-project/exposures).
```

Comment:
you should use a relative link here - this will let docusaurus verify the validity of the link! Think this should work:

dbt-cloud/dbt-cloud-api/metadata/metadata-overview

Can you make that change for all docs-internal link URLs in this diff?

Reply:
Also - let's change "arbitrary HTML" to "an iframe" :)